processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessPipeline.java

```diff
@@ -109,6 +109,7 @@ public void scheduleThenProcess(TaskRequest request) throws InterruptedException
         Timer timer = Utils.timer();
         CompletableFuture<Void> processResult;
         try (LoggingContext ignored = context.loggingContext()) {
+            scope.tracingProvider().unsafeProcessingStart(request.trace());
```

Comment:
This and below `xxReturn` call are better to be surrounded by try-catch with logging?
The implementation is blackbox from decaton core and while we expect them to never throw, if it happens, I feel like we shouldn't let decaton to abort just because of tracing function flaw.

Reply:
Point taken. Will do.